webrtc-c/canary/src/CloudwatchMonitoring.cpp

```diff
@@ -96,4 +95,82 @@ VOID CloudwatchMonitoring::pushICEHolePunchingDelay(UINT64 delay, StandardUnit u
     this->push(datum);
 }
 
+VOID CloudwatchMonitoring::pushOutboundRtpStats(Canary::POutgoingRTPMetricsContext pOutboundRtpStats)
+{
+    MetricDatum bytesDiscardedPercentageDatum, averageFramesRateDatum, nackRateDatum, retransmissionPercentDatum;
+
+    bytesDiscardedPercentageDatum.SetMetricName("Percentageframediscarded");
+    bytesDiscardedPercentageDatum.SetValue(pOutboundRtpStats->framesPercentageDiscarded);
+    bytesDiscardedPercentageDatum.SetUnit(StandardUnit::Percent);
+    bytesDiscardedPercentageDatum.AddDimensions(this->channelDimension);
+    this->push(bytesDiscardedPercentageDatum);
+
+    averageFramesRateDatum.SetMetricName("FramesPerSecond");
+    averageFramesRateDatum.SetValue(pOutboundRtpStats->averageFramesSentPerSecond);
+    averageFramesRateDatum.SetUnit(StandardUnit::Count_Second);
+    averageFramesRateDatum.AddDimensions(this->channelDimension);
+    this->push(averageFramesRateDatum);
+
+    nackRateDatum.SetMetricName("NackRate");
+    nackRateDatum.SetValue(pOutboundRtpStats->nacksPerSecond);
+    nackRateDatum.SetUnit(StandardUnit::Count_Second);
+    nackRateDatum.AddDimensions(this->channelDimension);
+    this->push(nackRateDatum);
+
+    retransmissionPercentDatum.SetMetricName("PercentageFramesRetransmitted");
+    retransmissionPercentDatum.SetValue(pOutboundRtpStats->retxBytesPercentage);
+    retransmissionPercentDatum.SetUnit(StandardUnit::Percent);
+    retransmissionPercentDatum.AddDimensions(this->channelDimension);
+    this->push(retransmissionPercentDatum);
+}
+
+VOID CloudwatchMonitoring::pushInboundRtpStats(Canary::PIncomingRTPMetricsContext pIncomingRtpStats)
+{
+    MetricDatum incomingBitrateDatum, incomingPacketRate, incomingFrameDropRateDatum;
+
+    incomingBitrateDatum.SetMetricName("IncomingBitRate");
+    incomingBitrateDatum.SetValue(pIncomingRtpStats->incomingBitRate);
+    incomingBitrateDatum.SetUnit(StandardUnit::Kilobits_Second);
```

Comment:
Is this supposed to be Bits_second instead?

Reply:
It is supposed to be Kilobits_second. I corrected the related computation